### PR TITLE
Doesn't display filter title when there is no filter types availables.

### PR DESCRIPTION
**Pull request by fanny**
#### What problem is this solving?

Sometimes, our filter does not contain filter options.
Currently, when it happens, the title of the filter type still shows up.
The correct behavior should be to only show filter titles when there are filter types available.
This pr fixes this.
[Clubhouse task](https://app.clubhouse.io/vtex-dev/story/11270/filters-should-not-show-their-title-when-there-are-no-options-available)

#### How should this be manually tested?

1. [Acess this workspace](https://filteroptions--storecomponents.myvtex.com)
2. as in my store-component workspace, i didn't find any department in that filters is empty, i set this https://github.com/vtex-apps/search-result/blob/3459761faa9dc62b1971e68c5d36d3ad90630893/react/FilterNavigator.js#L110 to an empty array

#### Types of changes

* [X] Bug fix (a non-breaking change which fixes an issue)

### CODE REVIEW

File: react/FilterNavigator.js [hunk 1]
```diff
@@ -137,28 +139,30 @@ const FilterNavigator = ({
...
+            </h5>
+          </div>
+          <SelectedFilters filters={selectedFilters} />
+          <AvailableFilters filters={filters} priceRange={priceRange} />
         </div>
-        <SelectedFilters filters={selectedFilters} />
-        <AvailableFilters filters={getFilters()} priceRange={priceRange} />
-      </div>
-    </div>
+        </div>
```
Comment (lbebber) [inline react/FilterNavigator.js]: formatting looks a bit weird

File: react/FilterNavigator.js [hunk 2]
```diff
@@ -107,6 +107,8 @@ const FilterNavigator = ({
     ].filter(Boolean)
   }
 
+  const filters = []
```
Comment (lbebber) [inline react/FilterNavigator.js]: should this be on this PR?

File: react/components/GalleryItem.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import { ExtensionPoint } from 'vtex.render-runtime'
+import { usePixel } from 'vtex.pixel-manager/PixelContext'
```
Comment (lbebber) [inline react/components/GalleryItem.js]: should this be part of this PR?

Comment (fanny) [reply react/components/GalleryItem.js]: when i made a rebase, this occurred, i don't know the reasons.

File: react/FilterNavigator.js (hunk 2 above)
Comment (fanny) [reply react/FilterNavigator.js]: Oh no, i'll fix.

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Doesn't display filter title when there is no filter types availables.
```
Comment (lbebber) [inline CHANGELOG.md]:
```suggestion
- Hides filters when there are no filters available.
```

**Pull request merged**